app/controllers/issues_controller.rb

```diff
@@ -10,25 +10,28 @@ def index
     issues_scope = Issue.where(status: params[:status])
     issues_scope = issues_scope.like(params[:filter]) if params[:filter]
     @status = params[:status]
+    feed_metatags
     smart_listing_create :issues,
                          issues_scope,
                          partial: 'issues/issue'
     @categories = Category.ordered_by_name
   end
 
   def new
+    set_meta_tags title: 'Створити нове звернення'
     @issue = current_user.issues.new
   end
 
   def show
     @issue = Issue.find(params[:id])
-    @relevant_issues = @issue.category.issues.where.not(id: @issue.id)
-                             .order('random()').limit(4)
+    issues_metatags
+    relevant_issues
```

Comment:
I don't understand what this method is doing.

Add "verb" ... sample: load_relevant_issues, find_...,